projects/004 - Finn/scripts/main.js

```diff
@@ -415,7 +416,9 @@ but where the name doesn't end with 'PatientId' then the ids will not be pseudon
           rowArr.push(value);
         }
       });
-      rowsToWrite.push(rowArr.join(',') + '\n');
+      // stringify will encode each value in the rowArr so that all values are complying with the CSV spec
+      const encodedCSVRow = stringify([rowArr]) + '\n';
```

Comment:
You mentioned possibly using this code for all the other projects. We can consider it, but I think it depends on what it does with strings. E.g. if we have an existing extract with a row like:
123,2020-01-01,UNPLANNED,Diabetes
Would this put quotes around the fields? Or does it only do that if there are commas? If the former then let's leave it because then if we redid an extract for someone, it would change. If it's the latter, then I'm happy for this to be the standard "main.js" file in all projects.

Reply:
Looking through the extracted tables, the csv stringify library only puts quotes when needed - so the example above would remain the same. 
I won't make the changes to the standard "main.js" file in this pull request - want to check it first with the 004 extract and verify with the data analysts. I will raise another PR for this.